src/main/webapp/college-info.html

```diff
@@ -1,21 +1,25 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Black Student Insight</title>
+    <title>College Info Page</title>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="style.css">
     <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
     <script src="script.js"></script>
   </head>
   <body>
     <header>
-      <h1>Black Student Insight</h1>
-      <input type="text" id="school-search">
-      <search-button onclick="loadSchoolInfo()">Search</search-button>
+      <a href="index.html"><h1>Black Student Insight</h1></a>
+      <input type="text" id="school-search" placeholder="Find your institution">
+      <button id="search-button" onclick="loadSchoolInfo()">Search</button>
     </header>
     <main>
-      <h1 id="school-name"></h1>
+      <div class="topnav">
+        <a class="active" href="college-info.html">School Info</a>
+        <a href="comments.html">Student Reviews</a>
+      </div>
 
+      <h1 id="school-name"></h1>
```

Comment:
This h1 will clash in hierarchy with the h1 in the header, because header isn't "sectioning content." Is it an h2 instead?

https://www.w3.org/TR/2010/WD-html5-20101019/content-models.html#:~:text=Sectioning%20content%20is%20content%20that%20defines%20the%20scope%20of%20headings%20and%20footers.&text=Each%20sectioning%20content%20element%20potentially,elements%20that%20are%20sectioning%20roots.